server/src/main/java/io/deephaven/server/util/GrpcServiceOverrideBuilder.java

```diff
@@ -266,8 +267,7 @@ public void invokeNext(
                     .onError(responseObserver)
                     .submit(() -> {
                         browserStream.get().onMessageReceived(request, streamData);
-                        responseObserver.onNext(null);// TODO simple response payload
-                        responseObserver.onCompleted();
+                        GrpcUtil.safelyComplete(responseObserver, null);
```

Comment:
I don't think this is OK. I believe Unary requests are required to yield a singular response.